### ISSUE TITLE
dunst does not make the summary bold if it contains '&'

### ISSUE THREAD

**Issue by belkka** [dunst-project/dunst#277]
The symbol '&' makes magic

```
$ notify-send 'summary' 'text'  # all right, summary is bold
$ notify-send 'summary with &' 'text'  # summary looks like 'text', it is not
bold
```

[http://i.imgur.com/BOVdS30.png](http://i.imgur.com/BOVdS30.png)

**Comment by LemonBoy** [comment]
[Here's](https://gist.github.com/LemonBoy/d0c197a32c4bd07463837c01d298a854) a patch :)

**Comment by tsipinakis** [comment]
Thanks for the patch @LemonBoy. But unfortunately it breaks notifications with markup in the summary or body.

Sometime later on I intend to do a full refactor of that part of the code in order to implement #279 which should also fix this. Until then, if you want, feel free to send a patch to fix this or submit a PR.

**Comment by tsipinakis** [comment]
Fixed with #279, added option to control how markup is handled `full` to parse markup, `strip` to strip it and `no` to handle everything as plain text.

The problem here is that `&` starts an entity, you can either make a matching rule to disable markup for this notification type, or replace all instances of `&` with `&amp;` in your notifications.
